src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -11,13 +11,40 @@ public function __construct($key, $usergroup)
     }
 }
 
+class PropertyKeyNotAllowedException extends \RuntimeException
+{
+    public function __construct($key)
+    {
+        parent::__construct(sprintf('Property key "%s" is reserved for internal use and overwritten when importing.', $key));
+    }
+}
+
 class Property
 {
+    /**
+     * Reserved property keys for internal use which would be overwritten when importing
+     *
+     * /image\d+/: Image URLs of type default.
+     * /thumbnail\d+/: Image URLs of type thumbnail.
+     * /ordernumber/: The products first exported ordernumber.
+     */
+    const RESERVED_PROPERTY_KEYS = [
```

Comment:
PHP version requirement should be set specifically to a version supporting array shorthand syntax, if using it. Also, it should be used consistently all over the project, if we want to go that way.

Reply:
Done